app/views/mentee_application_mailer/confirm_application.html.slim

```diff
@@ -18,3 +18,98 @@ p
   br
   i
     | The "Learn IT, Girl" team
+
+p
+  div
+    | Ps. Look again on you application below. If you'll notice anything that should be changed,
+    | please, contact us.
+  div
+    | Ps2. Especially, take a look on Programming Language, if it's the one that you choose.
+
+p
+  hr
+
+scss:
+  .application-content {
+    div strong { margin-right: 5px; }
+    h3 {padding-top: 20px;}
+    .paragraph {padding: 10px 0 10px 15px;}
+  }
+p
+  div.application-content
+    h3 Personal information
+    .paragraph
+      div
+        strong First name:
+        span #{@application.first_name}
+      div
+        strong Last name:
+        span #{@application.last_name}
+      div
+        strong Country:
+        span #{@application.country}
```

Comment:
I think we should display full name of the country (we're storing its code in db) `Carmen::Country.coded(@application.country).to_s` or `Carmen::Country.coded(@application.country).name`